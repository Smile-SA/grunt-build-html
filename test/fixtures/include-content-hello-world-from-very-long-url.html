<%= include('simple-skeleton', {
  title: 'My title',
  body: include('subfolder/subsubfolder/subsubsubfolder/very-deep-sububsubfolder/deeper/sodeep/content-hello-world')
}) %>