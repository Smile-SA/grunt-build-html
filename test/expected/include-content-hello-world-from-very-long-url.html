<!DOCTYPE html>
<html>
  <head>
    <title>My title</title>
  </head>
  <body>
    Hello world into very deep subfolder
  </body>
</html>
